android_tutorial_cv_bridge_CompressedImage_javacv/src/main/java/com/github/rosjava/android_extras/android_tutorial_cv_bridge/MainActivityCompressedJavacv.java

```diff
@@ -0,0 +1,148 @@
+package com.github.rosjava.android_extras.android_tutorial_cv_bridge;
```

Comment:
Ah, you're right, I'm very sorry @talregev . I'm crazy  :(  I'll merge like this.